indico/modules/events/controllers/api.py

```diff
@@ -25,3 +27,14 @@ def _process(self, event):
         elif not event.can_access(session.user):
             return jsonify(can_access=False)
         return EventDetailsForSeriesManagementSchema().jsonify(event)
+
+
+class RHEventCheckEmail(RHEventBase):
+    """Check a person's email when editing a person link."""
+
+    @use_kwargs({
+        'email': LowercaseString(required=True),
+    }, location='query')
+    def _process(self, email):
+        RHEventBase._process_args(self)
```

Comment:
that does not belong in `_process` - and it's called automatically anyway since `_process_args` is not being overridden here